File: week04-2.cpp
//week04-2
//leetcode 2529. maximum count of positive integer and negative integer
int maximumCount(int* nums, int numsSize) {

    int pos=0 ,neg=0;
    for (int i=0;i<numsSize;i++){
        if(nums[i]>0)pos++;
        if(nums[i]<0)neg++;
    }

    if(pos>neg) return pos;
    else return neg;
}

File: week04-2.py
# week04-2.py
# leetcode2529.maximum count of positive integer and negative integer
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0

        for i in range(len(nums)):
            if nums[i] > 0:
                pos += 1
            if nums[i] < 0:
                neg += 1

        return max(pos, neg)
